sorts/tree_sort_2.py

```diff
@@ -0,0 +1,131 @@
+"""
+
+tree_sort_2
+
+- makes use of the data structure Binary Tree to sort the list in O(nlogn) time.
+- Binary Search Tree [BST]
+  - For each node with value v
+    - All values in the left subtree are < v
+    - All values in the right subtree are > v
+  - No Duplicate values.
+- Logic:
+  - Build a Binary Search Tree
+  - Traverse the tree in increasing order.
+
+"""
+
+
+class Tree:
+    # Binary Search Tree Data Structure
+    # Constructor
+    def __init__(self, initval: int | str | float = None) -> None:
+        """
+        An empty left and right branches will be created for every value inserted,
+        to perform better in recursive methods
+        """
+        self.value = initval
+        if self.value:
+            self.left = Tree()
+            self.right = Tree()
+        else:
+            self.left = None
+            self.right = None
+        return
+
+    # Empty nodes are None valued
+    def isempty(self) -> bool:
+        """
+        Returns true if tree is empty else False
+        >>> isempty([])
```

Comment:
```suggestion
        >>> Tree().isempty()
```
It seems like `isempty` doesn't take any arguments, so why are you passing `[]`?

Reply:
Sure. I'll update. Thank you!